src/main/java/org/elasticsearch/action/search/SearchScrollRequest.java

```diff
@@ -39,6 +42,9 @@
     private String scrollId;
     private Scroll scroll;
 
+    private BytesReference source;
+    private boolean unsafe;
```

Comment:
The `unsafe` field can be removed here too, because we parse the source before we potentially fork to a different thread.